lib/services/spotify/save-user-info.ts

```diff
@@ -71,4 +71,6 @@ export const saveUserInfo = async ({
   if (isNewAccount) {
     await notifySlack(`🐝 New sign-up for *beecast*: ${data.display_name}`);
   }
+
+  return true;
```

Comment:
```suggestion
  return { isNewAccount } ;
```

Reply:
lol I forgot to fix this return true placement before commit :D